incubator/java-openliberty/image/project/invoke-appsody.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ -e /project/user-app/.appsody-nolocal ]
+then
+	BIN_TEMPLATE=1
+fi
+
+case $APPSODY_DEV_MODE in
+	run)
+		#if [ -e /project/user-app/.appsody-nolocal ]
+		if [ $BIN_TEMPLATE = 1 ]
```

Comment:
This produces an error message if BIN_TEMPATE isn't set.  

Maybe a better test would be:

`if [[ -f <file> ]]`

Reply:
this part is removed now, no longer needed. function moved to validate.sh